Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node
+
+WORKDIR /app
+
+ADD . /app
```

Comment:
If you don't need the magic of docker's `ADD` (ie extracting files, etc), you should use `COPY`.

Reply:
Changed.